### DISCUSSION TITLE
Is there a way to test database notifications?

### DISCUSSION THREAD

**Discussion by bzy107** [filamentphp/filament#10730] [Help]
Is there a way to test database notifications? 
There doesn't seem to be any documentation on it. https://filamentphp.com/docs/3.x/notifications/testing
![スクリーンショット 2024-01-10 10 12 26](https://github.com/filamentphp/filament/assets/28666304/c461037e-1f4e-4fd5-a4b2-59f9507cea38)

**Comment by leandrocfe** [comment]
Could you explain what would you like to test?

**Reply by bzy107** [reply]
I am using database notifications in the import feature, and in case of an error, an error link is displayed in the notification. I want to test that this link is being displayed.

thanks.
